src/lib/authToken.ts

```diff
@@ -109,6 +109,14 @@ const getDefaultAuthToken = ({ wallet }: DefaultAuthTokenArgs) => {
   return token;
 };
 
+export const getSha256AuthToken = ({ wallet }: DefaultAuthTokenArgs) => {
```

Comment:
I'd add a little warning comment here that this is likely not the one you want to use, and refer to the comment in `useKeyfileGenerator` for further details.

(One could argue that the main comment should live here, and the hook should refer to the lib for further details, but I don't think it matters all that much at the moment.)

Reply:
Good call, clarified the comments based on this feedback